src/Entity/Access/RatePlanAccessControlHandler.php

```diff
@@ -94,11 +94,14 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
     // If rate plan is a developer rate plan, and the assigned developer is
     // different from account, deny access.
     if ($rate_plan instanceof DeveloperRatePlanInterface) {
-      if ($developer = $rate_plan->getDeveloper()) {
```

Comment:
Variable assignment in a condition is dangerous and should be avoided.

Reply:
`Error: Call to a member function getEmail() on null in Drupal\apigee_m10n\Entity\Access\RatePlanSubscriptionAccessHandler->access() (line 102 of /mnt/files/local_mount/build/web/modules/contrib/apigee_m10n/src/Entity/Access/RatePlanSubscriptionAccessHandler.php) `

Reply:
This means that there was a developer, it had a developer-specific rate plan, but the developer was removed and the developer-specific rate plan remained in the system (Apigee) with a `null` developer reference.